specification/redis/resource-manager/Microsoft.Cache/stable/2022-06-01/redis.json

```diff
@@ -2670,12 +2670,12 @@
         "geoReplicatedPrimaryHostName": {
           "type": "string",
           "readOnly": true,
-          "description": "Fully qualified geo subdomain URL which points to geo-primary cache among the linked redis caches."
+          "description": "Fully qualified geo hostname which points to geo-primary cache among the linked redis caches."
         },
-        "geoReplicatedPrimaryHostNameDnsCName": {
+        "primaryHostName": {
           "type": "string",
           "readOnly": true,
-          "description": "DNS CName corresponding to geo subdomain URL which points to geo-primary cache among the linked redis caches. This value will be the actual DNS hostname for the Geo Primary cache."
+          "description": "DNS corresponding to geo-primary cache among the linked redis caches."
```

Comment:
Is the idea that this property will change over time to reflect geo-replication failover? If so could we change the description slightly, I'm thinking this would be clearer

"DNS name that resolves to the current geo-primary cache among the linked redis caches."

Reply:
Thanks Tim! This looks better. Updated.